processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -757,38 +772,48 @@ public int add(String value)
     }
 
     @Override
-    public int getSortedId(String value)
+    public DimLookup sort()
     {
-      return delegate.getSortedId(Strings.nullToEmpty(value));
+      return new NullValueConverterDimLookup(delegate.sort());
```

Comment:
There is no `getSortedId()` anymore which converts value into index. Instead we translates int[] directly.